common/changes/@bentley/schema-rpcinterface-common/rpcs-migration_2021-03-30-15-50.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you delete all of the schema-rpcinterface-* files? Seems they're still around after the rename.